include/deal.II/grid/grid_tools.h

```diff
@@ -1032,6 +1032,52 @@ namespace GridTools
   ( const MeshType                                                                    &mesh,
     const std::function<bool (const typename MeshType::active_cell_iterator &)> &predicate );
 
+  /**
+   * Compute a collection of bounding boxes so that all cells for which the given predicate is true, are
+   * completely enclosed in at least one of the bounding boxes.
+   *
+   * For each cell at a given refinement level containing active cells for which predicate is true,
+   * the function creates a bounding box of its children for which predicate is true.
+   *
+   * This results in a cover of these cells; the parameters @p allow_merge and @p max_boxes are used to
```

Comment:
Please specify exactly what you mean by "these". All active cells with the predicate that have a level >= the one specified?

Reply:
All active cells for which predicate it's true